emc-metalnx-services/src/main/java/com/emc/metalnx/services/irods/PermissionsServiceImpl.java

```diff
@@ -262,17 +267,10 @@ private boolean chmodDataObject(DataGridPermType permType, String path, String u
 			if (!inAdminMode) {
```

Comment:
There's a TODO comment on a very similar looking block above to "expand". Does the same need to be done here, or are they different?

Reply:
I don't believe so.  I think the cases are handled.  I've removed the TODO.